audio-worklet/src/bitcrusher-audio-worklet.js

```diff
@@ -40,7 +40,8 @@ registerProcessor('bitcrusher-audio-worklet',
   constructor(options) {
     super(options);
 
-    // Index_ and previousSample_ are globals to handle block transitions.
+    // Index_ and previousSample_ are member variables to handle block
```

Comment:
In a previous review I said the variable is `index_` and not `Index_`.  Perhaps use `|index_|` to make it clear?

`index_` isn't a great variable name and the comment doesn't help, but ok.